### ISSUE TITLE
Using Kirigami for the interface

### ISSUE THREAD

**Issue by ognarb** [blackaf/activitydesk#4]
Hi,

That do you think about using [Kirigami2](https://kde.org/products/kirigami/) for the user interface? I could implement this.

Best,

Carl

**Comment by jalcine** [comment]
I don’t have any opposition towards this! I opted for using stock Qt to reduce the number of dependencies needed to get the project built. I think, though, before this can be done, I’d have to clearly outline what pages are needed so your time isn’t wasted! How does that sound?

(Originally published at: https://v2.jacky.wtf/post/6b165cae-680a-4cb2-9bdc-cca82518bace)

**Comment by jalcine** [comment]
@ognarb hey are you still down to help out with this?

**Comment by ognarb** [comment]
Yes but I quite busy lately ;) I could start working on it next week.
